packager/util/util.go

```diff
@@ -52,7 +52,7 @@ func ParsePrivateKey(keyPem []byte) (crypto.PrivateKey, error) {
 		var pemBlock *pem.Block
 		pemBlock, keyPem = pem.Decode(keyPem)
 		if pemBlock == nil {
-			return nil, errors.New("invalid PEM block in private key file")
+			return nil, errors.New("invalid PEM block in private key file, make sure to use the right key type. Generate the key using: openssl ecparam -out priv.key -name prime256v1 -genkey")
```

Comment:
Maybe instead of inlining the openssl command, link to "step 2 of https://github.com/WICG/webpackage/tree/master/go/signedexchange#creating-our-first-signed-exchange", which includes the necessary command to regenerate the cert, too.

Alternatively, we could update our README or our wiki to include this information, and link to that.

Reply:
Done